website/docs/api-layers.md

```diff
@@ -971,11 +947,11 @@ https://github.com/explosion/thinc/blob/master/thinc/layers/chain.py
 Construct `n` copies of a layer, with distinct weights. For example,
 `clone(f, 3)(x)` computes `f(f'(f''(x)))`.
 
-| Argument    | Type                             | Description                        |
-| ----------- | -------------------------------- | ---------------------------------- |
-| `orig`      | <tt>Model[ArrayXd, ArrayXd]</tt> | The layer to copy.                 |
-| `n`         | <tt>int</tt>                     | The number of copies to construct. |
-| **RETURNS** | <tt>Model[ArrayXd, ArrayXd]</tt> | The composed model.                |
+| Argument    | Type               | Description                        |
+| ----------- | -------------------| ---------------------------------- |
+| `orig`      | <tt>Model</tt>     | The layer to copy.                 |
+| `n`         | <tt>int</tt>       | The number of copies to construct. |
+| **RETURNS** | <tt>Model</tt>, ... | The composed model.                |
```

Comment:
Not sure what the `...` signifies here?

Reply:
It's meant to signify the fact that more than one copy can be returned. I'm changing the text to make this clearer.

Reply:
But that's not right - it's always exactly one `Model` that is returned. It could just be that it contains multiple copies of the same layer, chained together.

Reply:
Got it!